docs/en/Configuration-Settings.md

```diff
@@ -81,9 +81,27 @@ $ cp conf/alluxio-site.properties.template conf/alluxio-site.properties
 Make sure that this file is distributed to `${ALLUXIO_HOME}/conf` on every Alluxio node (masters
 and workers) before starting the cluster.
 
-> Note that, `${ALLUXIO_HOME}/conf/alluxio-site.properties` are only loaded by Alluxio server
-> processes. Applications interacting with Alluxio service through Alluxio client will not load
-> these site property files, unless `${ALLUXIO_HOME}/conf` is on the application classpath.
+## Use Cluster Default
+
+Since v1.8, each Alluxio client can initialize the configuration with cluster-wise
+configuration values fetched from masters.
+To be specific, when different client applications like Alluxio Shell commands,
+Spark jobs, or MapReduce jobs connect to an Alluxio service,
+they will initialize their own Alluxio configuration properties with the default values
+supplied by the masters based on the `${ALLUXIO_HOME}/conf/alluxio-site.properties` files on
+masters. As a result, if masters put client-side settings (e.g., `alluxio.user.*`) or
+network transport settings (such as `alluxio.security.authentication.type`) in
+`${ALLUXIO_HOME}/conf/alluxio-site.properties`,
+these properties will become cluster-wise default values for new Alluxio clients
+unless these clients overwrite the same properties using the approaches described in
+[Configure Alluxio for Applications](Configuration-Settings.html#configure-applications),
+or specify property `alluxio.user.conf.cluster.default.enabled=false` to
+decline the cluster-wise default values.
+
+> Note that, before v1.8, `${ALLUXIO_HOME}/conf/alluxio-site.properties` file is only loaded by
+Alluxio server
+> processes and will be ignored by applications interacting with Alluxio service through Alluxio
```

Comment:
won't pre-v1.8 clients still use alluxio-site.properties if it's on their classpath?

Reply:
added.